backend/src/main/java/com/qualitydefectdetector/model/request/UserStoryRequestRow.java

```diff
@@ -0,0 +1,103 @@
+package com.qualitydefectdetector.model.request;
+
+import com.qualitydefectdetector.enums.UserStoryType;
+
+public class UserStoryRequestRow {
```

Comment:
I think it would be better if the class name was UserStory instead of UserStoryRequestRow.